platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/PlatformData.java

```diff
@@ -528,19 +528,25 @@ public long getMinimumGenerationNonAncient() {
      * @see PlatformState#getInfoString()
      */
     public String getInfoString() {
-        return new TextTable()
+        final String dataTable = new TextTable()
                 .setBordersEnabled(false)
                 .addRow("Round", round)
                 .addRow("Number of consensus events", numEventsCons)
-                .addRow("Consensus events running hash", hashEventsCons == null ? "null" : hashEventsCons.toMnemonic())
                 .addRow("Consensus timestamp", consensusTimestamp)
                 .addRow("Last timestamp", lastTransactionTimestamp)
                 .addRow("Rounds non-ancient", roundsNonAncient)
                 .addRow("Creation software version", creationSoftwareVersion)
                 .addRow("Epoch hash", epochHash == null ? "null" : epochHash.toMnemonic())
                 .addRow("Min gen info hash code", minGenInfo == null ? "null" : minGenInfo.hashCode())
                 .addRow("Events hash code", Arrays.hashCode(events))
+                .addRow(
+                        "Consensus events running mnemonic",
```

Comment:
```suggestion
                        "Consensus events running mnemonic: ",
```

Reply:
Adding colons to all values in the table, for consistency